site/src/components/TemplateStats/TemplateStats.tsx

```diff
@@ -26,77 +27,39 @@ export const TemplateStats: FC<TemplateStatsProps> = ({
   template,
   activeVersion,
 }) => {
-  const styles = useStyles()
-
   return (
-    <div className={styles.stats}>
-      <div className={styles.statItem}>
-        <span className={styles.statsLabel}>{Language.usedByLabel}:</span>
-
-        <span className={styles.statsValue}>
-          {formatTemplateActiveDevelopers(template.active_user_count)}{" "}
-          {template.active_user_count === 1
-            ? Language.developerSingular
-            : Language.developerPlural}
-        </span>
-      </div>
-      <div className={styles.statItem}>
-        <span className={styles.statsLabel}>{Language.buildTimeLabel}:</span>
-
-        <span className={styles.statsValue}>
-          {formatTemplateBuildTime(template.build_time_stats.start_ms)}{" "}
-        </span>
-      </div>
-      <div className={styles.statItem}>
-        <span className={styles.statsLabel}>
-          {Language.activeVersionLabel}:
-        </span>
-        <span className={styles.statsValue}>{activeVersion.name}</span>
-      </div>
-      <div className={styles.statItem}>
-        <span className={styles.statsLabel}>{Language.lastUpdateLabel}:</span>
-        <span className={styles.statsValue} data-chromatic="ignore">
-          {createDayString(template.updated_at)}
-        </span>
-      </div>
-      <div className={styles.statItem}>
-        <span className={styles.statsLabel}>{Language.createdByLabel}:</span>
-        <span className={styles.statsValue}>{template.created_by_name}</span>
-      </div>
-    </div>
+    <Stats>
+      <StatsItem
+        label={Language.usedByLabel}
+        value={
+          <>
+            {formatTemplateActiveDevelopers(template.active_user_count)}{" "}
+            {template.active_user_count === 1
```

Comment:
If you feel like messing with it, i18n has a way of handling this

Reply:
I will take a look. I have to move all the language stuff to i18n tho

Reply:
No worries if you want to put it off. I think there's an example in the `common` file.